pulsar-client-kafka-compat/pulsar-client-kafka/src/main/java/org/apache/kafka/clients/consumer/PulsarKafkaConsumer.java

```diff
@@ -304,8 +313,6 @@ public void unsubscribe() {
         });
     }
 
-    private static final int MAX_RECORDS_IN_SINGLE_POLL = 1000;
```

Comment:
we should still use the constant for `1000`, maybe renaming to `DEFAULT_MAX_RECORDS_IN_SINGLE_POLL`